### ISSUE TITLE
zone name in REST API

### ISSUE THREAD

**Issue by jpmens** [PowerDNS/pdns#2464]
PowerDNS 3.4.4 returns the following information via the REST API when enumerating zones:

```json
{
    "id": "=2E",
    "kind": "Master",
    "url": "/servers/localhost/zones/=2E"
```

According to the [documentation](https://doc.powerdns.com/md/httpapi/api_spec/), `id` is an

> opaque zone id (string), assigned by the Server. Do not interpret. Guaranteed to be safe for embedding in URLs.

The `url` obtained in a `GET ../zones` as well as `id` appear to be URL-encoded, and can be used to get the full zone data which in turn returns

```json
{
    "id": "=2E",
    "records": [
        {
...
    "url": "/servers/localhost/zones/=2E"
}
```

So, being ultra pedantic, what is the name of this zone if I'm not to interpret `id`? ;-)

In short: can we please have an unencoded zone name in there?

**Comment by zeha** [comment]
AFAICT, /zones and /zones/:id both already return a `name` field.
